cornflow/commands/commands.py

```diff
@@ -55,36 +69,59 @@ def get_options(self):
         return (
             Option("-e", "--email", dest="email", help="Service user email"),
             Option("-p", "--password", dest="password", help="Service user password"),
+            Option(
+                "-v",
+                "--verbose",
+                dest="verbose",
+                help="Verbose for the command. 0 no verbose, 1 full verbose",
+            ),
         )
 
-    def run(self, email, password):
+    def run(self, email=None, password=None, verbose=0):
         """
         Method to run the command and create the service user
 
         :param str email: the email for the admin user
         :param str password: the password for the admin user
+        :param int verbose: verboseof the command
         :return: a boolean if the execution went right
         :rtype: bool
         """
-        return create_user_with_role(email, password, "serviceuser", SERVICE_ROLE)
+        if email is None or password is None:
+            print("Missing required arguments")
```

Comment:
do we want to continue if both arguments are not set? I'm not sure to understand why put default arguments if we're going to check if they are None.

Reply:
But I still don't understand why the arguments have default values. Why do we allow default values if we're not going to let the function work without them. Shouldn't we just take out the default values (as it was before?)

Reply:
The default values were because PyCharm was raising an error that could be fixed assigning default values. See the image:

![image](https://user-images.githubusercontent.com/24654620/121642487-e02bcb80-ca90-11eb-9f09-0ae55ed43e36.png)

Thats why I added the default values, but I can take them out if you want